summarization/extractor_output_preprocessor.py

```diff
@@ -79,7 +86,9 @@ def _split_content(self):
         '''
         for content in self.content_list:
             if content.content_type == ContentType.TEXT:
-                if content.type == "title":
+                if (content.type in self.TITLE_CONTENT_TYPES
+                    or content.is_bold)\
+                        and len(content.text_string) < self.MAX_TITLE_LENGTH:
```

Comment:
```suggestion
                if (content.is_important_text() 
                and len(content.text_string) < self.MAX_TITLE_LENGTH)
```
create a method is_important_text in Text class.

Reply:
thanks for the suggestion. updated accordingly.